group_project_v2/utils.py

```diff
@@ -281,3 +283,7 @@ def add_resource(block, resource_type, path, fragment, via_url=False):
         action_parameter = loader.load_unicode(path)
 
     action(action_parameter)
+
+
+def get_block_content_id(block):
```

Comment:
@bradenmacdonald There should be at least three usages, and three usages is enough for me to factor it out. Unfortunately, it didn't fit into any existing mixin/base class, so I decided to put it here.